### DISCUSSION TITLE
click event don't work in some browser,like chorm.

### DISCUSSION THREAD

**Discussion by qilonelone** [nuxt/nuxt#21252] [Questions]
### Environment

Nuxi 3.5.1                                                                                                                                                        15:41:40
                                                                                                                                                                  15:41:41
RootDir: /Users/mingling/Documents/GitHub/chat-h5                                                                                                                 15:41:41
Nuxt project info: (copied to clipboard)                                                                                                                          15:41:41

------------------------------
- Operating System: Darwin
- Node Version:     v20.1.0
- Nuxt Version:     3.5.1
- Nitro Version:    2.4.1
- Package Manager:  yarn@1.22.19
- Builder:          vite
- User Config:      modules, i18n, tailwindcss
- Runtime Modules:  @nuxtjs/tailwindcss@6.7.0, @nuxtjs/eslint-module@4.1.0, nuxt-icon@0.4.1, @pinia/nuxt@0.4.11, @pinia-plugin-persistedstate/nuxt@1.1.1, @nuxtjs/i18n@8.0.0-beta.12, @nuxtjs/device@3.1.0
- Build Modules:    -

### Reproduction

It can work on a computer, but not on a phone. I originally thought it was my mistake. But when I use the website“ https://nuxt.com/“  testing this issue. When clicking on the menu, the click event does not work. Please provide assistance.

### Describe the bug

It can work on a computer, but not on a phone. I originally thought it was my mistake. But when I use the website“ https://nuxt.com/“  testing this issue. When clicking on the menu, the click event does not work. Please provide assistance.

**Comment by danielroe** [comment]
I am not familiar with 'chorm' but probably this means that the browser doesn't support modern JavaScript. You may need to update your `vite.build.target`: https://vitejs.dev/config/build-options.html#build-target.

**Reply by qilonelone** [reply]
thanks for your help, but i don't know how to set the option(vite.build.target) in 
```
export default defineNuxtConfig({
}）
```

**Reply by danielroe** [reply]
```ts
export default defineNuxtConfig({
  vite: {
    build: {
      target: 'es2015'
    }
  }
})
```

**Reply by qilonelone** [reply]
Sorry to bother you again, I have already configured it, but it still hasn't taken effect
